packages/debug-test/DebuggingFeatures.test.ts

```diff
@@ -9,35 +9,70 @@ import * as http from 'http';
 import {getDebugTargets, CDPDebugger} from './CDPDebugger';
 import {Metro} from './MetroAutomation';
 import * as path from 'path';
-import {PlaygroundDebugSettings, selectPackage} from './PlaygroundAutomation';
+import {
+  PlaygroundDebugSettings,
+  loadPackage,
+  primePackageComboBox,
+} from './PlaygroundAutomation';
 import {testLog} from './TestLog';
 import {formatDateTime} from './Utilities';
+import {pid} from 'process';
+import fs from '@react-native-windows/fs';
+//import {AutomationElement} from '@react-native-windows/automation';
```

Comment:
remove?

Reply:
Removed locally (to be published with next change iteration).